src/main/resources/static/templates/layout.html

```diff
@@ -154,12 +156,13 @@ <h1 id="logo"><a href="/"><strong>Project Reactor</strong></a></h1>
 
                     <span class="licence">Project Reactor regroups multiple open source
                         projects licensed under the Apache Software License 2.</span>
-                    © 2020 <a href="https://www.vmware.com/" class="pivotal">VMware, Inc.</a> or its affiliates. All rights reserved | <a
+                    © 2021 <a href="https://www.vmware.com/" class="pivotal">VMware, Inc.</a> or its affiliates. All rights reserved | <a
                         href="https://www.vmware.com/help/legal.html">Terms
                     of Use</a> |
-                    <a href="https://www.vmware.com/help/privacy.html">Privacy</a> |
-                    <a href="/security-policy">Security Policy</a> |
-                    <span id="teconsent"></span>
+                    <a href="https://www.vmware.com/help/privacy.html">Privacy</a> | 
+                    <a href="/security-policy">Security Policy</a> | 
+                    <a href="https://www.vmware.com/help/privacy/california-privacy-rights.html">Your California Privacy Rights</a> | 
+                    <a class="ot-sdk-show-settings">Cookie Settings</a>
```

Comment:
this doesn't seem to do anything, when running the site locally, even with the above `https:` modification (which would supposedly bring in the OT sdk) 🤔